### PR TITLE
feat:Add Organization(except built-in) admin permissions

### PR DESCRIPTION
**Pull request by forestmgy**
Fix: https://github.com/casdoor/casdoor/issues/1101

Add normal organization(except built-in) admin permission to c/u/r/d users and applications

### CODE REVIEW

File: object/user.go [hunk 1]
```diff
@@ -236,6 +236,24 @@ func getUser(owner string, name string) *User {
 	}
 }
 
+func GetUserByNameAndOwner(owner string, name string) *User {
```
Comment (hsluoyz) [inline object/user.go:239]: We already have this

File: controllers/base.go [hunk 2]
```diff
@@ -52,7 +52,7 @@ func (c *ApiController) IsGlobalAdmin() bool {
 		return false
 	}
 
-	return user.Owner == "built-in" || user.IsGlobalAdmin
+	return user.Owner == "built-in" || user.IsAdmin
```
Comment (hsluoyz) [inline controllers/base.go:55]: IsAdmin is org admin, not global admin

File: web/src/App.js [hunk 3]
```diff
@@ -496,6 +496,23 @@ class App extends Component {
         </Menu.Item>
       );
     }
+    if (Setting.isOrganizationAdminUser(this.state.account)) {
```
Comment (hsluoyz) [inline web/src/App.js:499]: Re-use exsiting menus

File: web/src/ApplicationListPage.js [hunk 4]
```diff
@@ -36,6 +36,7 @@ class ApplicationListPage extends BaseListPage {
       enableSigninSession: false,
       enableCodeSignin: false,
       enableSamlCompress: false,
+      organizationName: this.props.account.organizationName,
```
Comment (hsluoyz) [inline web/src/ApplicationListPage.js:39]: Why store it?

File: web/src/Setting.js [hunk 5]
```diff
@@ -419,7 +419,13 @@ export function isAdminUser(account) {
   if (account === undefined || account === null) {
     return false;
   }
-  return account.owner === "built-in" || account.isGlobalAdmin === true;
+  return account.owner === "built-in" ;
+}
+export function isOrganizationAdminUser(account) {
```
Comment (hsluoyz) [inline web/src/Setting.js:424]: isOrgAdminUser

File: web/src/Setting.js [hunk 6]
```diff
@@ -419,7 +419,13 @@ export function isAdminUser(account) {
   if (account === undefined || account === null) {
     return false;
   }
-  return account.owner === "built-in" || account.isGlobalAdmin === true;
+  return account.owner === "built-in" ;
```
Comment (hsluoyz) [inline web/src/Setting.js:422]: Don't change it

File: web/src/UserListPage.js [hunk 7]
```diff
@@ -373,7 +373,7 @@ class UserListPage extends BaseListPage {
     const field = params.searchedColumn, value = params.searchText;
     const sortField = params.sortField, sortOrder = params.sortOrder;
     this.setState({loading: true});
-    if (this.state.organizationName === undefined) {
+    if (this.state.organizationName === undefined && this.props.account.owner === "built-in") {
```
Comment (hsluoyz) [inline web/src/UserListPage.js:376]: isAdminUser

File: controllers/application.go [hunk 8]
```diff
@@ -55,6 +56,13 @@ func (c *ApiController) GetApplications() {
 		limit := util.ParseInt(limit)
 		paginator := pagination.SetPaginator(c.Ctx, limit, int64(object.GetApplicationCount(owner, field, value)))
 		applications := object.GetMaskedApplications(object.GetPaginationApplications(owner, paginator.Offset(), limit, field, value, sortField, sortOrder), userId)
+		for i := 0; i < len(applications); i++ {
+			key := applications[i].Organization
+			if key != organization && organization != "built-in" {
+				applications = append(applications[:i], applications[i+1:]...)
```
Comment (hsluoyz) [inline controllers/application.go:62]: Don't write like this

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: controllers/base.go (hunk 2 above)
Comment (forestmgy) [reply controllers/base.go:55]: IsGlobalAdmin is useless for all admin users(including "built-in" and other orgs), but IsAdmin is useful for orgs except "built-in",I use this function to determine if the current user is an administrator

**Comment by hsluoyz** [comment]
Replaced by: https://github.com/casdoor/casdoor/pull/1124

**Pull request closed without merging**